src/adapters/beachfront.js

```diff
@@ -14,7 +14,7 @@ function BeachfrontAdapter() {
     const bids = bidRequests.bids || [];
     bids.forEach(function(bid) {
       var bidRequest = getBidRequest(bid);
-      var RTBDataParams = prepareAndSaveRTBRequestParams(bid);
+      var RTBDataParams = prepareAndSaveRTBRequestParams(bidRequest);
```

Comment:
I changed it to `bidRequest` because it has the parsed width and height properties.